packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import crypto from 'crypto';
+import openBrowser from 'react-dev-utils/openBrowser';
+import { request } from '@octokit/request';
+import express, { Express, Request, Response } from 'express';
+
+const MANIFEST_DATA = {
+  default_events: ['create', 'delete', 'push', 'repository'],
+  default_permissions: {
+    contents: 'read',
+    metadata: 'read',
+  },
+  name: 'Backstage-<changeme>',
+  url: 'https://backstage.io',
+  description: 'Github App for Backstage',
+  public: false,
+};
+
+const FORM_PAGE = `
+<html>
+  <body>
+    <form id="form" action="ACTION_URL" method="post">
+      <input type="hidden" name="manifest" value="MANIFEST_JSON">
+      <input type="submit" value="Continue">
+    </form>
+    <script>
+      document.getElementById("form").submit()
+    </script>
+  </body>
+</html>
+`;
+
+type GithubAppConfig = {
+  appId: number;
+  apiUrl: string;
+  slug?: string;
+  name?: string;
+  webhookUrl?: string;
+  clientId: string;
+  clientSecret: string;
+  webhookSecret: string;
+  privateKey: string;
+};
+
+export class GithubCreateAppServer {
+  private baseUrl?: string;
+  private webhookUrl?: string;
+
+  static async run({ org }: { org: string }): Promise<GithubAppConfig> {
+    const encodedOrg = encodeURIComponent(org);
+    const actionUrl = `https://github.com/organizations/${encodedOrg}/settings/apps/new`;
```

Comment:
Maybe valuable to throw in a `// TODO` to note that this experimental command at this point does not support GitHub Enterprise? (not suggesting it should yet, but as it iterates, future iterations could either prompt for a GHE host or potentially call out to look for an app-config.yaml)

Reply:
GHE is definitely something we want to support in the CLI but it wasn't possible to post the manifest file unfortunately.

Reply:
@adamdmharvey I've added a comment by the command entrypoint. Please take a look :)

Reply:
merci @jhaals ! 😊